persistence/src/main/resources/META-INF/persistence.xml

```diff
@@ -18,8 +17,8 @@
       
       <!-- Statistics and Debugging -->
 
-      <property name="hibernate.format_sql" value="false" />
-      <property name="hibernate.show_sql" value="false" />
+      <property name="hibernate.format_sql" value="true" />
+      <property name="hibernate.show_sql" value="true" />
```

Comment:
Both hibernate.format_sql and hibernate.show_sql should be disabled

Reply:
I agree, revert these changes, otherwise looks fine.